server/node/call.go

```diff
@@ -99,15 +100,25 @@ func (c *Call) RowIter(ctx *sql.Context, r sql.Row) (sql.RowIter, error) {
 	}
 	if len(overloads) == 0 {
 		// We're going to assume that this is calling one of the few remaining Dolt stored procedures
-		sch, rowIter, _, err := c.Runner.Runner.QueryWithBindings(ctx, "", &vitess.Call{
-			ProcName: vitess.ProcedureName{
-				Name:      vitess.NewColIdent(c.ProcedureName),
-				Qualifier: vitess.NewTableIdent(c.SchemaName),
-			},
-			Params: c.originalExprs,
-		}, nil, nil)
-		c.cachedSch = sch
-		return rowIter, err
+		doltProcedureCallable, ok := functions.DoltProcedureCallables[c.ProcedureName]
```

Comment:
Looking at this a bit more, perhaps we should disallow Dolt procedures from being used through the Postgres `CALL` statement, since functionally all of the procedures should go through `SELECT procname()` since that's the closest analogue (`CALL` in Postgres isn't really used for returning values, which all of our Dolt procedures do for informational purposes). There will be some failing tests due to the change, but we can update those.

Reply:
I removed the use of procedures in `CALL` statements, and provide an explicit error for trying to use Dolt procedures with it.